src/SignalR/clients/csharp/Http.Connections.Client/src/Internal/WebSocketsTransport.cs

```diff
@@ -491,7 +491,6 @@ private async Task StartSending(WebSocket socket, bool ignoreFirstCanceled)
                 {
                     if (result.IsCanceled && !ignoreFirstCanceled)
                     {
-                        _logger.LogInformation("send canceled");
```

Comment:
?

Reply:
Was from debugging during the reconnect PR, got missed during cleanup.